### PR TITLE
Fix issue with multiple files upload

### PR DESCRIPTION
**Pull request by drakmail**
Correctly handle onChange event using fileOrGroup primise

### CODE REVIEW

**Comment by drakmail** [comment]
Saw this issue: https://github.com/uploadcare/angular-uploadcare/pull/3 (angularjs doesn't support two-way data binding for hidden inputs) and doesn't found any problems :confused: Changes from directive controller applies without problems and changes from directive's parent scope set from the element value applies without problems too.

**Comment by dmitry-mukhin** [comment]
@Morpho, any comments on this?

**Comment by dmitry-mukhin** [comment]
@drakmail, I would recommend to rename `fileOrGroup` to `data`, and `file` in `onChange` argument also to `data`

**Comment by drakmail** [comment]
@dmitry-mukhin no problem, done!

File: angular-uploadcare.js [hunk 1]
```diff
@@ -15,8 +15,9 @@ angular.module('ng-uploadcare', [])
         onWidgetReady: '&',
         onUploadComplete: '&',
         onChange: '&',
+        ngModel: '='
```
Comment (dmitry-mukhin) [inline angular-uploadcare.js]: I don't know angular :)
Is this the key change for the PR?

Comment (drakmail) [inline angular-uploadcare.js]: Just cosmetic change to drop dependency on $parse

Comment (dmitry-mukhin) [inline angular-uploadcare.js]: That's what I have meant.

**Comment by dmitry-mukhin** [comment]
Thanks!

**Pull request merged**